photon-core/src/main/java/org/photonvision/vision/pipeline/ObjectDetectionPipeline.java

```diff
@@ -52,6 +57,38 @@ protected void setPipeParamsImpl() {
         params.confidence = settings.confidence;
         params.nms = settings.nms;
         rknnPipe.setParams(params);
+
+        DualOffsetValues dualOffsetValues =
+                new DualOffsetValues(
+                        settings.offsetDualPointA,
+                        settings.offsetDualPointAArea,
+                        settings.offsetDualPointB,
+                        settings.offsetDualPointBArea);
+
+        SortContoursPipe.SortContoursParams sortContoursParams =
+                new SortContoursPipe.SortContoursParams(
+                        settings.contourSortMode,
+                        settings.outputShowMultipleTargets ? 5 : 1,
```

Comment:
what are these magic numbers

Reply:
Duplicated from other pipelines, but max targets in multi target mode.'I'll document